editor/src/messages/portfolio/document/utility_types/network_interface.rs

```diff
@@ -6590,49 +6659,6 @@ struct InputTransientMetadata {
 	// types: populated for each protonode after each
 }
 
-// TODO: Eventually remove this migration document upgrade code
-fn migrate_output_names<'de, D: serde::Deserializer<'de>>(deserializer: D) -> Result<Vec<String>, D::Error> {
```

Comment:
Is this moved elsewhere (that I can't find) or has it been fully removed? Unless its purpose has been obviated, we do still need these migrations.

Reply:
Removed. There is no need for these migrations, since the output name of a protonode is never visible. Also the output name should not be a type to begin with, as it is automatically set to be the type if the output name is an empty string.